precompile/allowlist/config.go

```diff
@@ -76,5 +85,28 @@ func (c *AllowListConfig) Verify(precompileconfig.ChainConfig) error {
 		addressMap[adminAddr] = AdminRole
 	}
 
+	if len(c.ManagerAddresses) != 0 && !upgrade.Disable && upgrade.Timestamp() != nil {
```

Comment:
do we need the disable check here? A disable config should not include manager addresses anyways, so I think we could remove `!upgrade.Disable`

Reply:
removed